Source Code compiling error

error: unknown type name 'SileroVadModelRknn'; did you mean 'SileroVadModel'

Last night i could compile the source code for iOS succesfully but now i get this error:
sherpa-onnx/sherpa-onnx/csrc/vad-model.cc:35:29: error: unknown type name 'SileroVadModelRknn'; did you mean 'SileroVadModel'?
   35 |     return std::make_unique<SileroVadModelRknn>(mgr, config);
      |                             ^~~~~~~~~~~~~~~~~~
      |                             SileroVadModel

--- a/sherpa-onnx/csrc/vad-model.cc
+++ b/sherpa-onnx/csrc/vad-model.cc
@@ -22,18 +22,22 @@
 namespace sherpa_onnx {
 
 std::unique_ptr<VadModel> VadModel::Create(const VadModelConfig &config) {
+#if SHERPA_ONNX_ENABLE_RKNN
   if (config.provider == "rknn") {
     return std::make_unique<SileroVadModelRknn>(config);
   }
+#endif
   return std::make_unique<SileroVadModel>(config);
 }
 
 template <typename Manager>
 std::unique_ptr<VadModel> VadModel::Create(Manager *mgr,
                                            const VadModelConfig &config) {
+#if SHERPA_ONNX_ENABLE_RKNN
   if (config.provider == "rknn") {
     return std::make_unique<SileroVadModelRknn>(mgr, config);
   }
+#endif
   return std::make_unique<SileroVadModel>(mgr, config);
 }
 

